imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
+  -File (((Get-UninstallRegistryKey 'Image Resizer for Windows').UninstallString -split '"')[1])
```

Comment:
`Get-UninstallRegistryKey` feels so wrong to me. What if the name is localized? It seems like this should be looked up using the upgrade code, not the name.